test/index.js

```diff
@@ -834,6 +834,66 @@ describe('Loveboat', () => {
             });
         });
 
+        it('passes options to the transform.', (done) => {
+
+            const server = new Hapi.Server();
+            server.connection();
+
+            const plugin = function (srv, options, next) {
+
+                srv.routeTransforms({
+                    name: 'post-to-patch',
+                    root: 'method',
+                    match: Joi.any().valid('post'),
+                    options: { after: ['get-to-post'] },
+                    handler: (method) => 'patch'
+                });
+
+                srv.loveboat({
+                    method: 'get',
+                    path: '/',
+                    handler: internals.boringHandler
+                }, [
+                    {
+                        name: 'patch-to-put',
+                        root: 'method',
+                        match: Joi.any().valid('patch'),
+                        options: { after: ['get-to-post', 'post-to-patch'] },
+                        handler: (method) => 'put'
+                    }
+                ]);
+
+                const table = srv.table()[0].table;
+
+                expect(table).to.be.an.array();
+                expect(table).to.have.length(1);
+                expect(table[0].method).to.equal('put');
+                expect(table[0].path).to.equal('/');
+
+                next();
+            };
+
+            plugin.attributes = {
+                name: 'plugin'
+            };
+
+            server.register([{
+                register: Loveboat,
+                options: {
+                    transforms: [{
+                        name: 'get-to-post',
+                        root: 'method',
+                        match: Joi.any().valid('get'),
+                        handler: (method) => 'post'
+                    }]
+                }
+            }, plugin], (err) => {
+
+                expect(err).to.not.exist();
+
+                done();
+            });
+        });
```

Comment:
This is just a straight copy of an above test, I think this could be written to include other things we'd want to use options for. What else did you have in mind for the options object?